app/models/manageiq/providers/kubernetes/container_manager_mixin.rb

```diff
@@ -72,6 +72,7 @@ def verify_ssl_mode
   end
 
   def connect(options = {})
+    options = options.dup
```

Comment:
Since we're touching the values here below anyway, you can actually do:

```ruby
sanitized_options = options.merge({
  :hostname => options[:hostname] || address,
  :port     => options[:port] || port,
  ...
})
```